NSS-Day-Cohort-49/tabloid-mvc-tabloid-mvc-cassettes
# HCR User Profile functionality

#### Changes Made
1. Added UserProfile views and Controllers/Repository/IRepository to add user profile functionality (see below).

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout hcr-userProfiles
    ```
2. Open a new Terminal tab (?T) and navigate to the server directory.
3. Test app functionality.
>click the User Profiles affordance
>see listing of active users
> test the functionality of the "Deactivate" and "Edit User Permissions" buttons
> select the "view deactivated users" affordance to view deactivated users.
> test functionality of reactivating users from the deactivated users listing.

4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
